src/pages/Organize/Folder/Create/index.js

```diff
@@ -14,12 +14,139 @@
+import { getAllFolders, createFolder } from "../../../../services/folders";
 
 const CreateFolder = () => {
-  return <div>Create Folder</div>;
+  const initialState = {
+    parentFolder: 1,
+    folderName: "",
+    folderDescription: "",
+  };
+  const initialFolderList = [
+    {
+      id: 1,
+      name: "Software Repository",
+      description: "Top Folder",
+      parent: null,
+    },
+  ];
+  const initialMessage = {
+    type: "success",
+    text: "",
+  };
+  const [createFolderData, setCreateFolderData] = useState(initialState);
+  const [folderList, setFolderList] = useState(initialFolderList);
+  const [loading, setLoading] = useState(false);
+  const [showMessage, setShowMessage] = useState(false);
+  const [message, setMessage] = useState(initialMessage);
+  const { parentFolder, folderName, folderDescription } = createFolderData;
+
+  const handleChange = (name) => (event) => {
+    setCreateFolderData({ ...createFolderData, [name]: event.target.value });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setLoading(true);
+    createFolder(createFolderData)
+      .then(() => {
+        setMessage({
+          type: "success",
+          text: "Successfully created the folder",
+        });
+      })
+      .catch((error) => {
+        setMessage({
+          type: "danger",
+          text: error.message,
+        });
+      })
+      .finally(() => {
+        setLoading(false);
+        setShowMessage(true);
+      });
+  };
+
+  useEffect(() => {
+    getAllFolders().then((res) => {
+      setFolderList(res);
+    });
+  }, []);
+
+  return (
+    <>
+      {showMessage && (
+        <Alert
+          variant={message.type}
+          onClose={() => setShowMessage(false)}
+          dismissible
+        >
+          <p>{message.text}</p>
+        </Alert>
+      )}
+      <div className="main-container my-3">
+        <h1 className="font-size-main-heading text-center">
+          Create a new Fossology Folder
+        </h1>
+        <br />
+        <div className="row">
+          <div className="col-12 col-lg-8">
+            <form>
+              <InputContainer
+                type="select"
+                name="parentFolder"
+                id="parentFolder"
+                onChange={handleChange("parentFolder")}
+                options={folderList}
+                value={parentFolder}
+              >
+                Select the parent folder:
+              </InputContainer>
+              <InputContainer
+                type="text"
+                name="folderName"
+                id="folderName"
+                onChange={handleChange("folderName")}
+                placeholder="Folder name"
+                value={folderName}
+              >
+                Enter the new folder name:
+              </InputContainer>
+              <InputContainer
+                type="text"
+                name="folderDescription"
+                id="folderDescription"
```

Comment:
@Aman-Codes please change the id's name because there may be chance of redundancy, can keep the id name's like `{folder}-{subFolder}-{property}`.
example: `organize-folder-create-description`